pkg/lang/frontend/starlark/config/config.go

```diff
@@ -212,3 +213,16 @@ func ruleFuncEntrypoint(thread *starlark.Thread, _ *starlark.Builtin,
 	ir.Entrypoint(argList)
 	return starlark.None, nil
 }
+
+func ruleFuncRepo(thread *starlark.Thread, _ *starlark.Builtin,
+	args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var url, description string
+
+	if err := starlark.UnpackArgs(ruleRepo, args, kwargs, "url?", &url, "description?", &description); err != nil {
```

Comment:
```suggestion
	if err := starlark.UnpackArgs(ruleRepo, args, kwargs, "url", &url, "description?", &description); err != nil {
```

`url` should be required.